### PR TITLE
New post: "React Native Navigation and ClojureScript"

### PR DESCRIPTION
**Pull request by svdo**

### CODE REVIEW

File: _posts/2019-04-02-react-native-navigation-and-clojurescript.md [hunk 1]
```diff
@@ -0,0 +1,494 @@
...
+### Versions
+
+Like I said in the beginning, future versions of any of the tools may break
+this guide. So it is only fair to mention which versions I was using for this:
+
+| Tool                    | Version                    |
+|-------------------------|----------------------------|
+| node                    | 11.10.1                    |
+| npm                     | 6.5.0-next.0               |
+| react                   | 16.8.3                     |
```
Comment (npalm) [inline _posts/2019-04-02-react-native-navigation-and-clojurescript.md]: For versions of react, react-native, react-native-navigation, shadow-cijs you can also refer to the package.json and yarn lock file. To make it even more friendly you can add a `package-lock.json` for the npm users as well.

File: _data/tags.yml [hunk 2]
```diff
@@ -99,4 +102,7 @@ rust:
   name: Rust
 xp:
   slug: xp
-  name: Extreme Programming
\ No newline at end of file
+  name: Extreme Programming
+shadow-cljs:
+  slug: shadow-cljs
+  name: shadow-cljs
```
[the remark this answers is not in the document]
Comment (svdo) [reply _data/tags.yml]: The author also writes it this way (shadow-cljs) so I just followed his lead.

File: _posts/2019-04-02-react-native-navigation-and-clojurescript.md [hunk 3]
```diff
@@ -0,0 +1,494 @@
+---
+layout:     post
+title:      "React Native Navigation and ClojureScript"
+subtitle:   "A beginner's guide to realizing proper navigation in a React Native
```
[the remark this answers is not in the document]
Comment (svdo) [reply _posts/2019-04-02-react-native-navigation-and-clojurescript.md]: Kind 'o did that on purpose :)

File: _posts/2019-04-02-react-native-navigation-and-clojurescript.md [hunk 4]
```diff
@@ -0,0 +1,494 @@
...
+header-img: "assets/2019-04-02-rnn-clojurescript/background.jpeg"
+tags:       [clojure, clojurescript, functional, react, react-native, shadow-cljs]
+---
+
+# React Native Navigation and ClojureScript
+
+*A beginner's guide to realizing proper navigation in a React Native application
+using ClojureScript and shadow-cljs*
+
+<p style="text-align: right">
```
[the remark this answers is not in the document]
Comment (svdo) [reply _posts/2019-04-02-react-native-navigation-and-clojurescript.md]: Nice that's good to know!

File: _posts/2019-04-02-react-native-navigation-and-clojurescript.md (hunk 1 above)
Comment (svdo) [reply _posts/2019-04-02-react-native-navigation-and-clojurescript.md]: Yes indeed; lock files are _really_ hard to read though, and I tried to put the selection here that I think actually matters for this post. I'm hesitating about the `package-lock.json`. On the one hand it's nice to `nom` users, on the other hand it is duplication that easily gets out of sync. I guess I'm just going to take the easy way out and leave it as is :)

**Comment by svdo** [comment]
Thanks for your review @npalm, very much appreciated! I decided to leave things as they are even though you have good comments. So as far as I'm concerned it can be merged&deployed.

**Pull request merged**